### DISCUSSION TITLE
Proxmox:  7.4-3 Kernel: 6.2.9-1

### DISCUSSION THREAD

**Discussion by Aterfax** [Aterfax/relax-intel-rmrr#35] [Announcements]
Built for Proxmox 7.4-3 via the Docker image.

Tested and works for passing a Quadro through on my DL380P.

Name: release.zip
Size: 126352769 bytes (120 MiB)
CRC32: 0DB3BF21
CRC64: 12EC704BF881B39B
SHA256: ec0550f417effb5fc7d8c4b5bd7cef9b37d8871a87ae286ff993dae2798ef9fc
SHA1: 4c43974a442eac1c5d8b1416d35e632049c9b1c9
BLAKE2sp: 5c4592abe984fa28eda7d95ed3b21ec3b8aab62f367ee2376bb32688fea2208b

This release and those going forward are signed with my GPG keys, see the following link for further details and the associated public key: https://github.com/Aterfax/relax-intel-rmrr/discussions/16

<hr /><em>This discussion was created from the release <a href='https://github.com/Aterfax/relax-intel-rmrr/releases/tag/v6.2.9-1'>Proxmox:  7.4-3 Kernel: 6.2.9-1</a>.</em>

**Comment by drizze99** [comment]
I just upgraded my kernel and it went to 6.2.11-1. Do you plan on patching that version?

**Reply by Aterfax** [reply]
I wasn't intending to release more often than monthly but I can take a look at testing and signing that kernel this weekend. 

For now, if you want to go ahead right away that kernel version was compiled automatically this weekend and you can grab it here: https://github.com/Aterfax/relax-intel-rmrr/actions/runs/4836421443

Edit: If you are concerned about reboots on a new unpatched kernel you should pin the latest patched one: https://pve.proxmox.com/wiki/Host_Bootloader#sysboot_kernel_pin

**Reply by drizze99** [reply]
Thanks! I am new to ProxMox and Linux in general. I got the kernel patched with the first link you posted. All verified and good to go. Now I just have to figure out how to get my GPU to passthrough my Hackintosh VM. I appreciate the reply and the work you've done.
